crates/tx5/src/ep3/sig.rs

```diff
@@ -311,7 +311,7 @@ impl Sig {
         }
     }
 
-    pub fn ban(&self, id: Id) {
+    pub fn close(&self, id: Id) {
```

Comment:
I created a separate `close` but it was identical to `ban` which doesn't actually ban, it's just called to close the connection when banning. Made most sense to me to just merge these

Reply:
Haha, yes this makes more sense :+1: